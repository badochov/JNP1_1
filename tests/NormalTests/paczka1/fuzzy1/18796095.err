Error in line 2: QPYfce9q2iR A95_ 40,8
Error in line 5: p87pY A956 59,5
Error in line 7: QPYfce9q2iR A956 171,0
Error in line 8: gleB A775 24,6
Error in line 15: S434 A556 155,3
Error in line 20: p87pY A613 28,3
Error in line 21: p87pY A956 79,7
Error in line 14: QPYfce9q2iR S434 94,2
Error in line 35: x7ev A775 77,6
