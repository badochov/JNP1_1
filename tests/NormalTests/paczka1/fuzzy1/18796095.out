dK4W A 111,8
S434 A 46,9 S 32,3
dK4W A 111,8 S 476,4
gleB A 106,3
p87pY A 1,9
x7ev A 212,9
S434 508,7
A613 106,3
A775 214,8
A956 158,7
p87pY A 1,9
S434 A 46,9 S 32,3
dK4W A 111,8 S 476,4
gleB A 106,3
p87pY A 1,9
x7ev A 352,1
zILvsNBw4 A 25,7
S434 508,7
A613 106,3
A775 379,7
A956 158,7
